src/test/java/com/cegeka/academy/repository/InterestTest.java

```diff
@@ -61,4 +67,33 @@ public void testFindByDescription(){
         assertThat(interestRepository.findByDescription("description1").get(0)).isEqualTo(interestRepository.findAll().get(0));
     }
 
+    @Test
+    public void testAddUserInterest(){
+        Interest interest1  = new Interest();
+        interest1.setName("interest1");
+        interest1.setDescription("description1");
+
+        User user = new User();
+        user.setEmail("gigi.gogaie@gmail.com");
```

Comment:
Incarca se definesti metoda @Before si @After, astfel incat sa-ti definesti intr-un singur loc obiectul de tip user si sa-l salvezi in baza de date, respectiv sa-l stergi dupa finalizarea metodei de test. Scopul acestei metode este de a test Interest nu User, ceea ce se  duce la concluzia ca presupunem ca avem un User in baza de date. Prin folosirea @Before si @After poti simula acest lucru